config/initializers/reload_api.rb

```diff
@@ -0,0 +1,11 @@
+if Rails.env.development?
```

Comment:
@mssola this reload API changes https://github.com/ruby-grape/grape#reloading-api-changes-in-development